functions-and-scope.js

```diff
@@ -54,25 +82,45 @@ const grades = [9, 8, 5, 7, 7, 4, 9, 8, 8, 3, 6, 8, 5, 6];
 // averageGrade([8, 9, 4, 6, 10]) geeft xxxx
 
 
+function averageGrade (grades) {
+    let sum = 0
+    for (let i = 0; i <grades.length; i++){
+    sum = sum + grades[i];
+    }
+    const average = sum / grades.length;
+    return average.toFixed(2);
+}
+console.log(averageGrade(grades));
+console.log(averageGrade([6,4,5]));
+console.log(averageGrade([8,9,4,6,10]));
+
 /* 2c: Afronden op twee decimalen */
 // Zorg ervoor dat het gemiddelde cijfer dat wordt teruggegeven uit de functie netjes wordt afgerond op twee decimalen.
 // Tip: Google is your best friend!
 
-
-
+// gebruik gemaakt van toFixed
 
 /* Bonusopdracht: hoogste cijfer */
 
 /* 3a: Script schrijven  */
 // Schrijf een script die op basis van de grades array (hierboven) checkt wat het hoogst behaalde cijfer is. Je mag hier geen bestaande methoden voor gebruiken. Schrijf de stapjes eerst uit en vraag jezelf de volgende dingen af:
-// * Hoe kan ik iedere waarde van de array langsgaan?
+// * Hoe kan ik iedere waarde van de array langsgaan? loop
 // * Op welke conditie moet ik checken?
 // * Hoe zorgt ik ervoor dat wanneer ik een cijfer tegenkom die aan de conditie voldoet, ik dit ergens kan opslaan?
 // Log het antwoord in de terminal.
 
 // ---- Verwachte uitkomst: 9
 
 
+    let highest = 0
```

Comment:
Het is geen goed voorbeeld, maar ik wil het je wel meegeven:
Wanneer je een array meekrijgt ([-1, -3, -2]) met alleen maar negatieve getallen, dan zal de functie altijd 0 teruggeven. Het is daarom verstandig om altijd de eerste waarde in de array te pakken:
```suggestion
    let highest = grades[0];
```